server/controllers/index.js

```diff
@@ -8,75 +8,81 @@ let currentUsers = [];
 
 /* Check if player is logged in */
 router.get('/auth', (req, res) => {
-    if (req.session.user) {
+    if (req.session.user) { /* The user session exists, authorised */
         res.status(200).json({ cookie: req.session.user });
-    } else {
-        res.sendStatus(401);
+    } else { /* The user session does not exist, unauthorised */
+        res.status(401).json({ message: "You need to log in with a username to enter the game."});
     }
 });
 
 /* Log in the user */
 router.post('/auth/login', (req, res) => {
     const user = currentUsers.find((user) => user.username === req.body.username);
-    if (!user) {
+    if (!user) { /* User does not already exist */
         req.session.user = req.body.username;            
```

Comment:
same here, you can destructure the username from req.body.. also means when building your 'user' object you can do:
`
const { username } = req.body;
const user = {
username
};
`

you dont need to do username: username